taf/updater/updater_pipeline.py

```diff
@@ -87,12 +87,27 @@ class UpdateState:
     temp_root: TempPartition = field(default=None)
 
 
+@attrs
+class UpdateOutputData:
```

Comment:
Update output also contains targets data. I'd recommend either renaming this (to something like AuthaCommitsData or adding targets data as a property as well)